src/MemoryBankController1.hpp

```diff
@@ -33,6 +28,8 @@ class MemoryBankController1 : public AMemoryBankController {
     ramData = ramPtr;
   }
 
+  friend class Accessor;
```

Comment:
Is this truly needed ? It seems that you just call public methods in your tests.

Reply:
indeed